src/nl/hannahsten/texifyidea/completion/LatexBibliographyReferenceProvider.kt

```diff
@@ -45,6 +45,14 @@ object LatexBibliographyReferenceProvider : CompletionProvider<CompletionParamet
                         }
                     }
                 }
-        result.withPrefixMatcher(CamelHumpMatcher(result.prefixMatcher.prefix, false)).addAllElements(lookupItems)
+        val before = result.prefixMatcher.prefix
+        val prefix =
+                if (before.contains(',')) {
+                    before.substring(before.lastIndexOf(',') + 1)
+                }
+                else {
+                    before
+                }
+        result.withPrefixMatcher(CamelHumpMatcher(prefix, false)).addAllElements(lookupItems)
```

Comment:
Not a  very elegant solution, but it works for now. I am not sure though whether maybe the parser should already impose more structure on required parameters, e.g., splitting them by comma.

Reply:
I was wondering about that a while ago as well. Maybe, but we need to check whether it could handle sufficiently many cases, because there may be LaTeX constructs in which the comma does not separate parameters (though I can't think of any right now, except if you would perhaps use a comma in a file name or something), since not all LaTeX commands do support separating arguments with commas (as the default is to have multiple required/optional parameters).

This was already on the project board under 'far future'. Might be that it stays there for a while, I don't think it's very important at the moment.

Reply:
I agree, it sounds like a major change and is probably not worth the hassle for now. I just wanted to hear your thoughts on this.